en/docs/references/how-to-use-statement-editor.md

```diff
@@ -1 +1,136 @@
-# Statement editor
\ No newline at end of file
+# Construct Statements using the Statement-Editor
+
+
+The Statement-Editor allows you to construct Ballerina statements depending on the use case you want to build. It is an intuitive visual editor that is easy to use even without an in-depth knowledge of the [Ballerina Programming Language](https://ballerina.io/).
+
+It provides the following:
+
+- Context-based suggestions
+
+- Ballerina libraries (standard libraries and language libraries)
+
+- Pre-defined expression templates
+
+- Documentation for functions and methods to easily configure parameters
+
+This guide walks you through the steps to build the following statement using the editing capabilities in the Statement-Editor. Later we will explore different use cases.
+
+```
+final string userIdMsg = "UserID : " + id.toString();
+```
+
+Let's add the above statement to the main method of the given sample code below
+
+```
+int id = 1234;
+ 
+public function main() returns error? {
+   // Statement will be added here
+}
+
+```
+
+
+To build the statement, follow these steps:
+
+1. On the low-code diagram, click **+** within the main method. Then click **Variable** in the **Add Constructs** pane that opens.
+
+    ![Add variable statement](../img/statement-editor/add-variable-statement.gif)
+
+    The Statement-Editor opens displaying the syntax for a variable statement with **`<add-expression>`** selected by default.
+
+2. Click the **var** type descriptor on the statement and click **string** on the **Suggestions** tab.
+
+    !!! info
+
+        The **Suggestions** tab provides contextual suggestions based on the selected item. The **Suggestions** tab currently lists type suggestions because you selected the type descriptor.    
```

Comment:
```suggestion
>**Info:** The **Suggestions** tab provides contextual suggestions based on the selected item. The **Suggestions** tab currently lists type suggestions because you selected the type descriptor.    
```